docs/source/x_ext.rst

```diff
@@ -1159,7 +1278,7 @@ A |coprocessor| is recommended (but not required) to follow the following sugges
 * Avoid using opcodes that are reserved or already used by RISC-V International unless for supporting a standard RISC-V extension.
 * Make it easy to change opcode assignments such that a |coprocessor| can easily be updated if it conflicts with another |coprocessor|.
 * Clearly document the supported and required parameter values.
-* Clearly document the supported and required interfaces (the memory (request/response) interface and memory result interface are optional).
+* Clearly document the supported and required interfaces.
```

Comment:
This requirement no longer makes sense now as all other interfaces than the memory itfs are mandatory.

Reply:
Ok. When working through the document, only then i realized that the compressed interface is not considered optional. In my opinion, it is inherently optional as it is only possible to implement if you have compressed encodings. Maybe something to disucss next meeting.

Reply:
fixed in 88e71d8